### PR TITLE
Fix issue with cattr not structuring from config file properly

### PR DESCRIPTION
**Pull request by josephaxisa**
- minor formatting changes in line with the google style guide
- added pytest-cov as a test coverage library as a dev dependency

### CODE REVIEW

File: python/looker/rtl/api_methods.py [hunk 1]
```diff
@@ -33,7 +34,7 @@ def get(self,
             structure: sr.TStructure,
             path: str,
             query_params: Optional[MutableMapping[str, str]] = None
-            ) -> sr.TDeserializeReturn:
+           ) -> sr.TDeserializeReturn:
```
Comment (joeldodge79) [inline python/looker/rtl/api_methods.py]: funny, our separate usages of yapf produce different results! might be this issue ["YAPF would at times not produce deterministic formatting..."](https://news.ycombinator.com/item?id=17155048) I'm curious to try out [Black](https://github.com/psf/black)

File: python/looker/rtl/api_settings.py [hunk 2]
```diff
@@ -33,5 +44,9 @@ def configure(cls,
         section = section or cfg_parser.sections()[0]
 
         cfg = dict(cfg_parser[section])
-        settings: ApiSettings = cattr.structure(cfg, cls)
+
+        converter = cattr.Converter()
+        converter.register_structure_hook(
+            bool, lambda string, _: _convert_bool(string))
+        settings: ApiSettings = converter.structure(cfg, cls)
```
Comment (joeldodge79) [inline python/looker/rtl/api_settings.py]: 🚀 thanks for figuring this out!

File: python/looker/rtl/auth_token.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 class AuthToken():
     """Used to instantiate or check expiry of an AccessToken object"""
+
```
Comment (joeldodge79) [inline python/looker/rtl/auth_token.py]: again, yapf differences, wonder if we're somehow each using different defaults (I don't see a .style.yapf file anywhere so I presume we're both using the defaults but then why the differences?)

File: python/looker/rtl/serialize.py [hunk 4]
```diff
@@ -7,7 +7,6 @@
 from typing import Callable, Dict, List, Type, Union
 
 import cattr
-
```
Comment (joeldodge79) [inline python/looker/rtl/serialize.py]: huh, yapf really shouldn't kill this line. there should be a single blank line between groupings of imports (stdlib, 3rd-party, own-lib/app)

File: python/looker/rtl/transport.py [hunk 5]
```diff
@@ -22,24 +21,13 @@ class HttpMethod(enum.Enum):
...
 @attr.s(auto_attribs=True)
 class TransportSettings:
     """Basic transport settings.
     """
     base_url: str
     api_version: str = '3.1'
-    # TODO: this isn't working ... we want "False" to be False
-    # the only way to get false now is leave "verify_ssl" blank
-    verify_ssl: bool = attr.ib(converter=_convert_bool, default=True)
+    verify_ssl: bool = True
```
Comment (joeldodge79) [inline python/looker/rtl/transport.py]: 😌 so much cleaner!

File: python/looker/rtl/user_session.py [hunk 6]
```diff
@@ -19,6 +19,7 @@ class UserSessionError(Exception):
 class UserSession():
     """UserSession to provid automatic authentication
     """
+
```
Comment (joeldodge79) [inline python/looker/rtl/user_session.py]: why doesn't my yapf do this? it's pep8 compliant which I think is the default style

File: python/looker/rtl/user_session.py [hunk 7]
```diff
@@ -70,10 +71,8 @@ def _login(self, user_id: Optional[str]) -> ml.AccessToken:
         self._reset()
 
         serialized = urllib.parse.urlencode({
-            'client_id':
```
Comment (joeldodge79) [inline python/looker/rtl/user_session.py]: ok, I officially hate my `yapf`

File: python/looker/rtl/auth_token.py (hunk 3 above)
Comment (josephaxisa) [reply python/looker/rtl/auth_token.py]: Interesting. I use `yapf --style=google --in-place`, is that what you are doing?

**Pull request merged**